mockall/src/lib.rs

```diff
@@ -1388,7 +1388,7 @@ pub use mockall_derive::concretize;
 ///
 /// When mocking a generic struct's implementation of a generic trait, use the
 /// same name for their generic parameters.  For example, if you wanted to mock
-/// `Rc`, do
+/// [`Rc`], do
```

Comment:
Likewise, I don't think this needs to be a link.  It's not really important for this example that users know exactly what Rc does.